sdk/include/opentelemetry/sdk/metrics/exemplar/reservoir.h

```diff
@@ -72,7 +72,7 @@ class ExemplarReservoir
       size_t size,
       std::shared_ptr<ReservoirCellSelector> reservoir_cell_selector,
       std::shared_ptr<ExemplarData> (ReservoirCell::*map_and_reset_cell)(
-          const common::OrderedAttributeMap &attributes));
+          const opentelemetry::common::OrderedAttributeMap &attributes));
```

Comment:
This seems under SDK, not API namespace.

Reply:
Done.